packages/fast-css-editor-react/src/position/position.style.ts

```diff
@@ -1,89 +1,107 @@
 import { ComponentStyles } from "@microsoft/fast-jss-manager";
-import { toPx } from "@microsoft/fast-jss-utilities";
+import { background800, foreground800 } from "../editor.constants.style";
+import {
+    applyInputStyle,
+    applySelectInputStyles,
+    applySelectSpanStyles,
+} from "../editor.utilities.style";
 
 export interface CSSPositionClassNameContract {
     cssPosition?: string;
-    cssPosition_selectContainer?: string;
-    cssPosition_selectContainer_select?: string;
+    cssPosition_control?: string;
+    cssPosition_select?: string;
     cssPosition_input?: string;
     absoluteInput_row?: string;
-    absoluteInput_row_center?: string;
-    absoluteInput_row_center__activeTop?: string;
-    absoluteInput_row_center__activeBottom?: string;
-    absoluteInput_row_center__activeLeft?: string;
-    absoluteInput_row_center__activeRight?: string;
+    absoluteInput?: string;
+    absoluteInput_centerRow?: string;
+    absoluteInput_centerRow__activeTop?: string;
+    absoluteInput_centerRow__activeBottom?: string;
+    absoluteInput_centerRow__activeLeft?: string;
+    absoluteInput_centerRow__activeRight?: string;
+    absoluteInput_centerRowContent?: string;
 }
 
 const styles: ComponentStyles<CSSPositionClassNameContract, {}> = {
-    cssPosition: {},
-    cssPosition_selectContainer: {
+    cssPosition: {
+        fontSize: "11px",
```

Comment:
I wonder if this actually should be 10 or 12? Since this is matching the form generator, this is fine here; more curious from @Falkicon if 11px is accurate as the value or if it should be 10 or 12? 

@janechu no need to make the change here, question is more from a design standpoint so I can file a bug.